save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -54,14 +56,36 @@ class UserDetailsService(
      * @param name
      * @throws NoSuchElementException
      */
-    fun updateAvatarVersion(name: String) {
+    fun updateAvatarVersion(name: String): String {
         val user = userRepository.findByName(name).orNotFound()
-        var version = user.avatar?.substringAfterLast("?")?.toInt() ?: 0
+        var version = if (user.avatar?.find { it == '?' } != null) {
```

Comment:
Suggestion to use more Kotlin solution: `user.avatar?.find { ... }?.let { ... } ?: ...`

Reply:
Yes, I thought about it. Because it was me who invented this rule in diktat. 

But in this particular case the code becoming less readable in ideomatic format